asv/plugins/summarylist.py

```diff
@@ -52,8 +52,7 @@ def publish(cls, conf, repo, benchmarks, graphs, revisions):
                     pretty_name = benchmark['pretty_name']
 
                 if idx is not None:
-                    pretty_name = '{0}({1})'.format(pretty_name,
-                                                    ", ".join(benchmark_param))
+                    pretty_name = f'{pretty_name}({", ".join(benchmark_param)})'
```

Comment:
This is fine, but I don't think we're improving readability that much. I forgot to mention in the issue I think, but for non-trivial code, would be better to first assign to a variable, and then have the f-string, but with the variable name, not with the whole expression. In this case it's not that complex a simple join, but having quotes and commas on the expression, makes the f-string quite unreadable in my opinion.

Reply:
Thanks i had thought about this,makes total sense